docs/guide/migration-from-0.6-to-1.0.md

```diff
@@ -0,0 +1,199 @@
+# Migrating from 0.6 to 1.0
+
+To upgrade your HyperFormula version from 0.6.x to 1.0.x, follow this guide.
+
+## Step 1: Change your license key
+
+Change your HyperFormula license key.
```

Comment:
Sounds scary 😄 Maybe we start with an explanation that only agpl license changed and non commercial was removed so our customers are not triggered from line one?

Reply:
@wojciechczerniak - right ;) Done: 21f936a